### ISSUE TITLE
Add short -c flag for --config

### ISSUE THREAD

**Issue by meridius** [rvben/rumdl#575]
### Problem or motivation

I run `rumdl` via [Megalinter](https://megalinter.io/beta/descriptors/markdown_rumdl/), which has a bug (as of v9.4.0) since it passes the `-c` flag instead of `--config` to `rumdl`, which `rumdl` (as of v0.1.74) doesn't recognize. 

I know this is primarily a compatibility problem of the MegaLinter, that's why I've created issue there <https://github.com/oxsecurity/megalinter/issues/7608>.

### Proposed solution

It'd be great if you could add a short `-c` flag as an alternative to the already existing long `--config`.

### Alternatives considered

I currently have to bypass the MegaLinter bug by specifically telling it the correct flag using this config in `.mega-linter.yml`:

```
# MarkdownLint passes -c but Rumdl expects --config
MARKDOWN_RUMDL_COMMAND_REMOVE_ARGUMENTS: "-c"
MARKDOWN_RUMDL_ARGUMENTS: [--config]
```

### Feature category

CLI / configuration

### Example markdown

```markdown

```

### Contribution

- [ ] I would be willing to submit a PR to implement this feature

**Comment by rvben** [comment]
Thanks @meridius! Added in v0.1.75 — `-c` now works as a short alias for `--config`.
